Mathlib/MeasureTheory/MeasurableSpace/Card.lean

```diff
@@ -25,147 +25,193 @@ indeed generates this sigma-algebra.
 -/
 
 
-universe u
+universe u v
 
 variable {α : Type u}
 
-open Cardinal Set
-
--- Porting note: fix universe below, not here
-local notation "ω₁" => (Ordinal.toType <| Cardinal.ord (aleph 1))
+open Cardinal Ordinal Set MeasureTheory
 
 namespace MeasurableSpace
 
 /-- Transfinite induction construction of the sigma-algebra generated by a set of sets `s`. At each
 step, we add all elements of `s`, the empty set, the complements of already constructed sets, and
-countable unions of already constructed sets. We index this construction by an ordinal `< ω₁`, as
-this will be enough to generate all sets in the sigma-algebra.
+countable unions of already constructed sets.
+
+We index this construction by an arbitrary ordinal for simplicity, but by `ω₁` we will have
+generated all the sets in the sigma-algebra.
 
 This construction is very similar to that of the Borel hierarchy. -/
-def generateMeasurableRec (s : Set (Set α)) (i : (ω₁ : Type u)) : Set (Set α) :=
-  let S := ⋃ j : Iio i, generateMeasurableRec s (j.1)
+def generateMeasurableRec (s : Set (Set α)) (i : Ordinal) : Set (Set α) :=
```

Comment:
This could perfectly well be indexed by an `Ordinal.{0}`, but I decided having the universe generality didn't cause any harm.